openx/prebid-mobile-ios
# Mobile 7104 test app for podspec

- added pod specs for the Rendering-related modules
- added a test app for rendering pod specs

We need the follow changes to the release documentation: https://docs.google.com/document/d/1k7mWpx7JE9vPykSZviWkn9VvZoqPztBo-2suZPnyuqk/edit#heading=h.prmiy424qbk0

In the section **Cocoapods Release**
--
Release the Rendering module podspec file:
1. `pod lib lint PrebidMobileRendering.podspec --verbose --allow-warnings`
2. `pod trunk push PrebidMobileRendering.podspec --verbose --allow-warnings`
3. Wait 20-30 minutes as podspec publishing requires some time
4. Release podspecs for the event handlers and adapters:
4.1 `pod lib lint PrebidMobileMoPubAdapters.podspec --verbose --allow-warnings`
4.2 `pod trunk push PrebidMobileMoPubAdapters.podspec --verbose --allow-warnings`
4.3 `pod lib lint PrebidMobileGAMEventHandlers.podspec --verbose --allow-warnings`
4.4 `pod trunk push PrebidMobileGAMEventHandlers.podspec --verbose --allow-warnings`
--

In the section **Test Cocoapods integration**:
--
Use the _TestPodspecModules_ application (tools/TestPodspecModules) for testing cocoapods integration with the rendering module and event handlers.
1. Set the app's version to the value of the SDK version
2. Update pods: `pod update`
3. Run the app. Check that the 'Project version' and the 'Rendering Module version' have the same values. Or you can just run _TestPodspecModulesUITests_.
--
